src/Compilers/.editorconfig

```diff
@@ -15,6 +15,9 @@ dotnet_diagnostic.RS0101.severity = none
 # RS0102: Braces must not have blank lines between them
 dotnet_diagnostic.RS0102.severity = none
 
+# IDE0170: Prefer extended property pattern
+dotnet_diagnostic.IDE0170.severity = suggestion
```

Comment:
only compiler change.  moving this back to a suggestion instead of a warning as i believe the compiler prefers not to enable automated style analyzers?  @jcouv  to confirm?

Reply:
Note: if compiler is ok turning this on, i'm happy to remove this line and fix up the remaining hits in the compiler layer.